tools/test_inference_models.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# ==============================================================================
+#  Copyright 2018 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+
+from build_utils import *
+
+
+def main():
+    '''
+    Builds TensorFlow, ngraph, and ngraph-tf for python 3
+    '''
+    parser = argparse.ArgumentParser(formatter_class=RawTextHelpFormatter)
+
+    parser.add_argument(
+        '--model_location',
+        help="Location of the model directory\n",
+        action="store")
+
+    parser.add_argument(
+        '--python_location',
+        help=
+        "Location of Python executable (whether virtual environment or system)\n",
+        action="store")
+
+    parser.add_argument(
+        '--artifacts_dir',
+        type=str,
+        help="Location of the artifacts\n",
+        action="store")
+
+    arguments = parser.parse_args()
+
+    #-------------------------------
+    # Recipe
+    #-------------------------------
+
+    # Save the current location
+    pwd = os.getcwd()
+
+    if not (arguments.artifacts_dir):
+        raise "Need to provide artifacts directory"
+
+    if not (arguments.python_location):
+        raise "Need to provide the location of Python directory"
+
+    if not (arguments.model_location):
+        raise "Need to provide the location of models directory"
+
+    # Check to make sure that there is TensorFlow installed and will work
+    # TODO
```

Comment:
Do you mean something like this in the TODO? 
```
import tensorflow as tf
assert (tf.__version__ == '1.13.1')
```

Reply:
Should we put `test_inference_models.py` under `tools` or `test`?

Reply:
No. We need to find out what version of TF was used to build NG and what version of TF this one is.

Reply:
@sayantan-nervana 
>Should we put test_inference_models.py under tools or test

No. I am planning to move these to a new ci directory. for now this is temp.